Formula/i/ircd-irc2.rb

```diff
@@ -4,6 +4,11 @@ class IrcdIrc2 < Formula
   url "http://www.irc.org/ftp/irc/server/irc2.11.2p3.tgz"
   version "2.11.2p3"
   sha256 "be94051845f9be7da0e558699c4af7963af7e647745d339351985a697eca2c81"
+  # The `:cannot_represent` is for:
+  # * International Business Machines, Inc. license in ircd/nameser_def.h
```

Comment:
See https://gitlab.com/fedora/legal/fedora-license-data/-/issues/53.
https://github.com/bminor/glibc/commit/ae49a7b29acc184b03c2a6bd6ac01b5e08efd54f

We should consider patching this code to use libc instead to avoid this.

Other license seems OK - it's a variation of `HPND` that might be worth upstreaming to SPDX (or just call it close enough to `HPND` and move on?)

Reply:
> We should consider patching this code to use libc instead to avoid this. (rm `nameser_def.h` and inreplace `"nameser_def.h"` to `<arpa/nameser_compat.h>`

Will probably require some more work than this, at least on macOS. I think `nameser.h` is needed (w/ `BIND_8_COMPAT` defined) for things like `ns_t_a`, but this causes conflicts in `res_comp.c` (e.g. `ns_name_ntop`). Would require rewriting parts of that too.

Reply:
If `res.c` is the only place that uses `T_` defines, then we maybe scope the extra `nameser.h` include to that file.

But yeah it perhaps gets to a point that maybe it's time to phase out this formula in favour of `ircd-hybrid`. The install count is really low.

Reply:
`#if 0`-ing out the duplicate functions and linking to `libresolv` worked. Can consider deprecating if causes any issues.